engine/access/rest/websockets/controller.go

```diff
@@ -135,9 +186,20 @@ func (c *Controller) writeMessages(ctx context.Context) error {
 			if err := c.conn.SetWriteDeadline(time.Now().Add(WriteWait)); err != nil {
 				return fmt.Errorf("failed to set the write deadline: %w", err)
 			}
-			err := c.conn.WriteJSON(msg)
-			if err != nil {
-				return fmt.Errorf("failed to write message to connection: %w", err)
+
+			if err := c.conn.WriteJSON(message); err != nil {
+				return err
+			}
+
+			// Update lastMessageSentAt
+			lastMessageSentAt = time.Now()
+
+		case <-inactivityTicker.C:
+			// Periodically check inactivity
```

Comment:
I'd change comment to sth like 
"If there are no active subscriptions and the connection is inactive for some time, we return"

Reply:
[bbc23ea](https://github.com/onflow/flow-go/pull/6817/commits/bbc23ea87e373d21a6618dd1be34bbb6ab0e8ca7)